amundsen_application/static/js/ducks/search/sagas.ts

```diff
@@ -90,16 +90,16 @@ export function* filterWatcher2(): SagaIterator {
 export function* filterWorker2(action: any): SagaIterator {
   const state = yield select(getSearchState);
   const { pageIndex = 0, resourceType, term = '' } = action.payload;
-  yield put(searchResource(term, resourceType, pageIndex));
+  yield put(searchResource("filter", term, resourceType, pageIndex));
   updateSearchUrl({ term, filters: state.filters, resource: resourceType, index: pageIndex }, false);
 };
 
 export function* inlineSearchWorker(action: InlineSearchRequest): SagaIterator {
   const { term } = action.payload;
   try {
     const [tableResponse, userResponse] = yield all([
-      call(API.searchResource, 0, ResourceType.table, term),
-      call(API.searchResource, 0, ResourceType.user, term),
+      call(API.searchResource, 0, ResourceType.table, term, {}, "automatic"),
+      call(API.searchResource, 0, ResourceType.user, term, {}, "automatic"),
```

Comment:
Not a fan of using `automatic`. Would prefer `InlineSearch.REQUEST` here.

Reply:
I'm going to make this one resolve to "inline". For context, the types of searched I considered "automatic" were inline searches and the new search that happens under the hood when the user clears the search term. I am just going to call everything exactly what it is. Considering the following:
```
export enum SearchType {
  CLEAR_TERM = 'clearTerm',
  FILTER = 'filter',
  INLINE_SEARCH = 'inline',
  LOAD_FROM_URL = 'loadFromUrl',
  PAGINATION = 'pagination',
  SEARCH_BAR = 'searchbar',
}
```